utils/ratelimits.py

```diff
@@ -0,0 +1,101 @@
+from datetime import datetime, timedelta
+import json
+
+from flask import abort, request
+
+
+class KeyExpired(KeyError):
```

Comment:
Why inherit KeyError? Would `Exception` not be better to inherit?

Reply:
Actually I don't know why I implemented that since it's never used when raised and raising it is not needed in this application

Reply:
Resolved